File: swagger/models/get_time_cards_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse
from .time_card_event import TimeCardEvent

class GetTimeCardsResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		time_cards
		Information about time card entries, ordered by staff ID.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'TimeCards': ['time_cards', TimeCardEvent],
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'time_cards': 'TimeCards',
		}

	pagination_response = None
	time_cards = None

File: swagger/Models/GetTimeCardsResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetTimeCardsResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse 
	 */
	public $paginationResponse;
	/**
	 * Information about time card entries, ordered by staff ID.
	 * 
	 * @var TimeCardEvent[] 
	 */
	public $timeCards;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class, null],
			'TimeCards' => ['timeCards', 'array', TimeCardEvent::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => ['PaginationResponse', PaginationResponse::class, null],
			'timeCards' => ['TimeCards', 'array', TimeCardEvent::class],
			];
	}
}
